src/bond.ts

```diff
@@ -568,57 +623,264 @@ export class BondContract extends Contract {
     return setContractStateTx;
   }
 
+  /**
+   *Builds a transaction to blacklist addresses.
+   * @param senderAddress - The address of the sender.
+   * @param addresses - An array of addresses to be removed from the blacklist.
```

Comment:
addresses description not accurate.

```suggestion
   * @param addresses - An array of addresses to be added as accepted callers for `bond` endpoint
```